tests/impl.cpp

```diff
@@ -7587,7 +7587,18 @@ result_t test_mm_round_sd(const SSE2NEONTestImpl &impl, uint32_t i)
 
 result_t test_mm_round_ss(const SSE2NEONTestImpl &impl, uint32_t i)
 {
-    return TEST_UNIMPL;
+    const float *_a = impl.mTestFloatPointer1;
+    const float *_b = impl.mTestFloatPointer1;
+    float d0 = bankersRounding(_b[0]);
+    float d1 = _a[1];
+    float d2 = _a[2];
+    float d3 = _a[3];
+
+    __m128 a = do_mm_load_ps(_a);
+    __m128 b = do_mm_load_ps(_b);
+    __m128 c = _mm_round_ss(a, b, _MM_FROUND_CUR_DIRECTION);
```

Comment:
The `bankers rounding` is the same as the rounding mode `_MM_FROUND_TO_NEAREST_INT`.
`_MM_FROUND_CUR_DIRECTION` means it uses the rounding mode which is set by `_MM_SET_ROUNDING_MODE()` before.

The way to fix it is to add
```
_MM_SET_ROUNDING_MODE(_MM_ROUND_NEAREST);
```
or to replace it with
```
__m128 c = _mm_round_ss(a, b, _MM_ROUND_NEAREST);
```

However, the testing should cover the four different rounding mode mentioned in the [document](https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_round_ss).